canonical-websites/jaas.ai
# Add community results

## Done

- Add the community search results

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Do some searches, they should have a community results.

## Details

- Fixes: #55.
